sdk/test/unit/plugin/utils.test.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  InvalidDigitsValueError,
+  ValueOutOfRangeError,
+  decodeRatio,
+  encodeRatio,
+} from '../../../src';
+
+describe('plugin/utils', () => {
+  describe('encodeRatio', () => {
+    it('Should encode a bigint from a float and the number of digits', () => {
+      const inputs = [
+        {float: 0.5, digits: 1, out: BigInt(5), error: null},
+        {float: 1, digits: 4, out: BigInt(10000), error: null},
+        {float: 0.25555, digits: 2, out: BigInt(26), error: null},
```

Comment:
the naming is a bit confusing in general in these tests
some are named `const inputs` & some are named `const tests`
some object has `output` and some has `out`

i suggest creating a type of needed or at least follow a consistent structure:

test = { input, output, error }